bigframes/session.py

```diff
@@ -930,7 +931,7 @@ def _read_pandas(
 
         job_config = bigquery.LoadJobConfig(schema=schema)
         job_config.clustering_fields = cluster_cols
-        job_config.labels = {"bigframes-api": api_name}
+        job_config.labels = {"bigframes-io-api": api_name}
```

Comment:
Why do we need a different key for this?

Reply:
Change this back by removing the io.